proto/provenance/marker/v1/marker.proto

```diff
@@ -92,6 +93,16 @@ enum MarkerStatus {
   MARKER_STATUS_DESTROYED = 5 [(gogoproto.enumvalue_customname) = "StatusDestroyed"];
 }
 
+// NetAssetValue defines a marker's net asset value
+message NetAssetValue {
+  // price is the complete value of the asset's volume
+  cosmos.base.v1beta1.Coin price = 1 [(gogoproto.nullable) = false];
+  // volume is the volume of the assets
```

Comment:
Can we reword this? "volume is the total number of the assets" or something similar?

Reply:
See if it make better sense now?